dockerhub/teamcity-server/README.md

```diff
@@ -44,7 +48,35 @@ To achieve that, you can pass an additional
 started with an alternative `server-https-proxy.xml` configuration file which enables HTTPS options.
 
 Alternatively, you can use a custom Tomcat configuration (see below).
-                       
+
+
+#### Configuring HTTPS Access to TeamCity Server
+
+If a TeamCity Server uses the [HTTPS connection](https://www.jetbrains.com/help/teamcity/using-https-to-access-teamcity-server.html), it transmits encrypted traffic through port `443` by default. 
+
+For security reasons, some operating systems impose restrictions
+on binding "privileged" ports (typically, ports below 1024) for non-root users, such as user 1000. As a result, port `443` can be unavailable for TeamCity Containers launched under _user 1000_.
```

Comment:
I'd rather say access or using. You can suggesfully bind the port, but your application won't be able to use it

Reply:
>As a result, port `443` can be unavailable for TeamCity Containers launched under _user 1000_.

The port will be available for the container, but not for the process inside the container.

Reply:
fixed